rsocket/statemachine/StreamsFactory.cpp

```diff
@@ -25,9 +28,30 @@ StreamsFactory::StreamsFactory(
               : 2 /*streams initiated by the server MUST use
                     even-numbered stream identifiers*/) {}
 
+void subscribeToErrorFlowable(
+    Reference<yarpl::flowable::Subscriber<Payload>> responseSink) {
+  yarpl::flowable::Flowables::error<Payload>(
+      folly::make_exception_wrapper<std::runtime_error>(
```

Comment:
Nit: `folly::make_exception_wrapper<Exn>` is unnecessary.  `exception_wrapper` can be explicitly constructed from any subclass of `std::exception`.